lib/spack/spack/cmd/common/arguments.py

```diff
@@ -185,7 +185,8 @@ def installed_specs():
 def yes_to_all():
     return Args(
         '-y', '--yes-to-all', action='store_true', dest='yes_to_all',
-        help='assume "yes" is the answer to every confirmation request')
+        help="""assume "yes" is the answer to every confirmation request
+excluding --no-checksum.""")
```

Comment:
I think this is still a little vague.  What we want to get across is that if saying `--yes-to-all` is not *quite* sufficient to build non-interactively -- you also need `--no-checksum`.

So something like this might be clearer:

> do not ask for confirmation (to run completely non-interactively, also specify --no-checksum)

Reply:
Thanks for the advice. I added a part of the line you recommended.